src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+@import '/src/styles/blocks/variables';
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  font-family: $main-font-family;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__photo {
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: contain;
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: rgba(6, 11, 53, 1);
+    text-align: left;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: rgba(97, 96, 112, 1);
+    margin: 0 0 16px;
+    line-height: 14px;
+  }
+
+  &__price-text {
+    color: rgba(97, 96, 112, 1);
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 16px;
+  }
+
+  &__price-amount {
+    color: rgba(6, 11, 53, 1);
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+    margin: 0;
+    line-height: 18px;
+  }
+
+  &__button {
+    background-color: rgba(0, 172, 220, 1);
+    color: rgba(255, 255, 255, 1);
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    width: 166px;
```

Comment:
You have defined a fixed width for the button. It would be better to use a width of 100% and then control the width of the button by controlling the width of its parent element. This will make your layout more flexible and responsive.